### ISSUE TITLE
Signing fails in docker builder:wine after updating versions

### ISSUE THREAD

**Issue by BHuys** [electron-userland/electron-builder#4741]
electron-builder
22.4.0
electron-updater
4.0.14
target
windows NSIS 

After updating electron from 6.0.12 to 8.0.3 and electron-builder from 20.44.4 to 22.4.0 I get an error when signing my application in the builder docker-box (:wine)

/root/.cache/electron-builder/winCodeSign/winCodeSign-2.6.0/linux/osslsigncode: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory

• downloading     url=https://github.com/electron-userland/electron-builder-binaries/releases/download/winCodeSign-2.6.0/winCodeSign-2.6.0.7z size=5.6 MB parts=1
  • downloaded      url=https://github.com/electron-userland/electron-builder-binaries/releases/download/winCodeSign-2.6.0/winCodeSign-2.6.0.7z duration=326ms
  • signing         file=/pipeline/source/ocularium-electron-release/win-unpacked/project-name.exe certificateFile=/tmp/t-061nXs/0.p12

has something changed in setting up this build flow over versions?

**Comment by BHuys** [comment]
nevermind, there was an issue with our CI cache.

**Comment by curiduck** [comment]
Had it solved?

**Comment by NeuralMayu** [comment]
Now the problem is appearing,what should i do?

**Comment by NeuralMayu** [comment]
![image](https://user-images.githubusercontent.com/82007499/186835919-072c25fa-2749-4937-8ae8-0dfa6807878d.png)

**Comment by BHuys** [comment]
Haven't worked with Electron in quite a while and can't really remember how I solved it. 

However, I notice there are special characters in your paths, maybe changing those exe names to names with only normal characters might solve the issue as some build tools might fail over that.

**Comment by dawangraoming** [comment]
> Now the problem is appearing,what should i do?

I got the same error, did u fix it? 🥲

**Comment by matheusteodoro01** [comment]
I have the same problem, it started yesterday
![image](https://user-images.githubusercontent.com/60862196/186905533-d9b28036-d84f-4ce4-b41f-5c95653485f3.png)

**Comment by dawangraoming** [comment]
There was an update pushing 21 hours ago, I'm not sure if this is the problem.

But I tried to switch to [16-wine-05.22](https://hub.docker.com/layers/builder/electronuserland/builder/16-wine-05.22/images/sha256-aa80803eea2b64ef675cc78443e50021cb2011fb120d9838f49c7714b8ff8120?context=explore) 

this error has been resolved.

@matheusteodoro01 @NeuralMayu

**Comment by dawangraoming** [comment]
<img width="1247" alt="image" src="https://user-images.githubusercontent.com/7522275/186911507-6049adc2-0b70-4ca1-a440-28808a0a2e89.png">

Ummmm, but I got another error now.

**Comment by matheusteodoro01** [comment]
Here it works using the electronuserland/builder:16-wine-mono-05.22! Thanks @dawangraoming

**Comment by flaushi** [comment]
i am facing the same problem as @NeuralMayu

**Comment by ejfrancis** [comment]
@flaushi the updated docker image that @matheusteodoro01 mentioned (electronuserland/builder:16-wine-mono-05.22) fixes it. you just need to update your build to use the new image

**Comment by flaushi** [comment]
Yes, sorry for not answering, I went from the latest `electronuserland/builder:wine-chrome` to `electronuserland/builder:16-wine-chrome-05.22`, which solved my problem.

However: this is no update, this is a downgrade.

electronuserland/builder:16-wine-chrome-05.22 was released on May 30, 2022. 
4 days ago, [16-wine-chrome-08.22](https://hub.docker.com/layers/builder/electronuserland/builder/16-wine-chrome-08.22/images/sha256-e1eb64b5404f2df103e32305fd1ecc0f51fff1ad74a102647ed7ac26d810474f?context=explore) was released, which produces the error. 

So I guess there will be a bugfix necessary, the current version is not able to sign windows executables.
